.dir-locals.el

```diff
@@ -0,0 +1,4 @@
+;;; Directory Local Variables
+;;; For more information see (info "(emacs) Directory Variables")
+
+((nil . ((fill-column . 80))))
```

Comment:
Actually, why are we setting it to 80? 100 is more customary in Lisp, due to long-ish names.

Anyway, that's a discussion for another time